bin/ingest_single_footprint.sh

```diff
@@ -48,7 +47,6 @@ cat << EOF
     min_zoom=greatest(0, approximate_zoom - 4),
```

Comment:
Not that it _really_ matters, but the 2 `UPDATE`s could get merged together using a CTE (`WITH blah AS ( select ... approximate_zoom ) UPDATE ...`